app/javascript/packs/pages/ProviderPage.jsx

```diff
@@ -32,15 +40,21 @@ const useStyles = makeStyles(theme => ({
   description: {
     color: "#747474",
     fontWeight: 100
+  },
+  button: {
+    marginTop: theme.spacing(6),
+    borderRadius: "25px",
+    color: "white",
+    backgroundColor: "#0f2e32"
   }
 }));
 
 const GET_PROVIDER = gql`
-  query Provider($id: ID) {
-    providers(filters: { id: $id }, orderBy: { direction: ASC, sort: CITY }) {
+  query Provider($id: ID!) {
+    provider(id: $id) {
       edges {
```

Comment:
With the change I suggested, you can remove `edges { node {` and just do

```
provider(id: $id) {
  id
  ...
```

Reply:
Running into issues with this refactor and I don't want to hold us up any longer. Definitely saving this for a future commit!